src/50_Sample_AMPs/eCommerce_Sample.html

```diff
@@ -0,0 +1,168 @@
+    <!-- #### Setup -->
+    <link rel="canonical" href="https://ampbyexample.com/sample_amps/ecommerce_sample/">
+    <!--
+      Import the libraries. 
+    -->
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    
+    <script async custom-element="amp-carousel" src="https://cdn.ampproject.org/v0/amp-carousel-0.1.js"></script>
+    <script async custom-element="amp-accordion" src="https://cdn.ampproject.org/v0/amp-accordion-0.1.js"></script>
+    <script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
+    <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
+    <script async custom-element="amp-social-share" src="https://cdn.ampproject.org/v0/amp-social-share-0.1.js"></script>
+
+    <style amp-custom>
+
+      ul.product-list {
+       list-style-type: none;
+       padding-left: 0;
+     }
+
+     ul.product-list a {
+       cursor: pointer;
+     }
+   </style>
+
+   <script type="application/ld+json">
+    {
+      "@context": "http://schema.org/",
+      "@type": "Product",
+      "name": "Nexus 6P",
+      "image": "http://www.ampbyexample.com/img/gold_nexus.png",
+      "description": "Gold Nexus",
+      "mpn": "925872",
+      "brand": {
+        "@type": "Phone",
+        "name": "Nexus"
+      },
+      "aggregateRating": {
+        "@type": "AggregateRating",
+        "ratingValue": "4.4",
+        "reviewCount": "88"
+      },
+      "offers": {
+        "@type": "Offer",
+        "priceCurrency": "USD",
+        "price": "500",
+        "priceValidUntil": "2020-11-05",
+        "itemCondition": "http://schema.org/UsedCondition",
+        "availability": "http://schema.org/InStock",
+        "seller": {
+          "@type": "AMPByExample",
+          "name": "AMPByExample"
+        }
+      }
+    }
+  </script>
+
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+
+  <!-- #### Create a product listing with AMP -->
+    <!-- Use `amp-accordion` to allow user to learn more about your product. Use `amp-list` with `amp-mustache` to retrieve a list of related products as images. 
```

Comment:
Have a look at how other samples a structured. Each usage of an AMP component is annotated with the corresponding explanation. In this case, add explanations for why you are using amp-list, amp-accordion, etc when each component is used.